lib/bolt/config/options.rb

```diff
@@ -475,6 +477,12 @@ module Options
           _plugin: true,
           _example: { "cleanup" => false, "tmpdir" => "/tmp/bolt" }
         },
+        "lxd" => {
+          description: "A map of configuration options for the LXD transport",
```

Comment:
```suggestion
          description: "A map of configuration options for the LXD transport. The LXD transport is "\
                               "experimental and may include breaking changes between minor versions.",
```
I want "experimental" to show up in the reference docs.

Reply:
Oo good call.